docker-compose.yml

```
@@ -43,7 +43,7 @@ services:
       - "443:443"
     volumes:
       - ./frontend/static_vol:/var/www/html
-      - ${CERTS}:/etc/nginx/ssl
+      - ${CERTS}:/etc/nginx/ssl:ro
```

suggestion、編集前の行を含めてはいけないのですね。。
こちらならcommit suggestionできるはず
```suggestion
      - ${CERTS}:/etc/nginx/ssl
```